oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/components/BaseFormLayout/BaseFormLayout.jsx

```diff
@@ -32,6 +38,35 @@ export const BaseFormLayout = ({ formikProps }) => {
   const {
     formConfig: { custom_fields: customFields },
   } = useFormConfig();
+  // on chrome there is an annoying issue where after deletion you are redirected, and then
+  // if you click back on browser <-, it serves you the deleted page, which does not exist from the cache.
+  // on firefox it does not happen.
+  useEffect(() => {
+    const handleUnload = () => {};
+
+    const handleBeforeUnload = () => {};
+
+    window.addEventListener("unload", handleUnload);
+    window.addEventListener("beforeunload", handleBeforeUnload);
+
+    return () => {
+      window.removeEventListener("unload", handleUnload);
+      window.removeEventListener("beforeunload", handleBeforeUnload);
+    };
+  }, []);
+
+  const urlHash = window.location.hash.substring(1);
+  let errorData;
+  if (urlHash) {
+    const decodedData = decodeUnicodeBase64(urlHash);
+    errorData = JSON.parse(decodedData);
+    window.history.replaceState(
+      null,
+      null,
+      window.location.pathname + window.location.search
```

Comment:
This replacing of window history feels hacky, why do we need to do this in case of any random error?

Reply:
If thats the case, I would go for navigating to #, using `navigate('#');`.
This way it won't change URL and query params and it will allow you to reset the hash.